### PR TITLE
[gms-1505] readme for erc1155 preset

### PR DESCRIPTION
**Pull request by jasonzwli**

### CODE REVIEW

File: contracts/token/erc1155/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+[Read more On the Threat Model](../../../audits/202312-threat-model-preset-erc1155.md)
+
+## preset/draft-ImmutableERC1155
+
+The draft-ImmutableERC1155 contract is the pre-release version of the Immutable's preset 1155 contract. It has been internally audited and is ready to be tested. Please note it is not yet supported by the rest of the Immutable ecosystem. The contract contains all external facing interfaces that are needed to interact(read and write) with deployed ERC1155 collections. 
+
+## abstract/ERC1155Permit
+
+This is an abstract contract that is used as an ancestor to the preset 1155. It contains the permit feature for the preset, allowing token owners to give approval and permission to a secure and trusted actor. This action is gasless for the token owner but will require gas from the approved operator. Please note that using permits in an ERC1155 collections exposes all the tokens owned by an address to the approved operator. Please double check to make sure the operate is secure and trusted.
```
Comment (allan-almeida-imtbl) [inline contracts/token/erc1155/README.md:13]: > Please double check to make sure the operate is secure and trusted.

Should `operate` be `operator` or `operation`?

File: contracts/token/erc1155/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# ERC1155 contracts
+
+The ImmutableERC1155 contracts allows clients to mint multiple different tokens with different token ids within the same collection. The contract features methods to allow for minting multiples of multiple token ids to simplify the minting flow and reduce gas costs. 
+
+[Read more On the Threat Model](../../../audits/202312-threat-model-preset-erc1155.md)
+
```
Comment (allan-almeida-imtbl) [inline contracts/token/erc1155/README.md:6]: What about a link to the standard here for more info: https://eips.ethereum.org/EIPS/eip-1155

File: contracts/token/erc1155/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+This is an abstract contract that is used as an ancestor to the preset 1155. It contains the permit feature for the preset, allowing token owners to give approval and permission to a secure and trusted actor. This action is gasless for the token owner but will require gas from the approved operator. Please note that using permits in an ERC1155 collections exposes all the tokens owned by an address to the approved operator. Please double check to make sure the operate is secure and trusted.
+
+## abstract/IERC1155Permit
+
+Provides the required interface for ERC1155Permit
+
+## abstrat/ImmutableERC1155Base
+
+This is another abstract contract is used as an ancestor to the preset 1155. It implements many of the internal methods for the preset that are not directly public facing for everyday uses. It contains functionalities to update royalties, set URIs and manage collection roles.
```
Comment (allan-almeida-imtbl) [inline contracts/token/erc1155/README.md:21]: > This is another abstract contract is used as an ancestor to the preset 1155.

grammar suggestions:

This is another abstract contract that is used as an ancestor to the preset 1155.
This is another abstract contract used as an ancestor to the preset 1155.

File: contracts/token/erc1155/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## abstract/ERC1155Permit
+
+This is an abstract contract that is used as an ancestor to the preset 1155. It contains the permit feature for the preset, allowing token owners to give approval and permission to a secure and trusted actor. This action is gasless for the token owner but will require gas from the approved operator. Please note that using permits in an ERC1155 collections exposes all the tokens owned by an address to the approved operator. Please double check to make sure the operate is secure and trusted.
+
+## abstract/IERC1155Permit
+
+Provides the required interface for ERC1155Permit
+
+## abstrat/ImmutableERC1155Base
```
Comment (allan-almeida-imtbl) [inline contracts/token/erc1155/README.md:19]: typo abstrat

**Review by allan-almeida-imtbl — APPROVED** [review]

**Review by ipekt — APPROVED** [review]

**Comment by allan-almeida-imtbl** [comment]
outdated by already updated docs: https://github.com/immutable/contracts/blob/main/contracts/token/erc1155/README.md

**Pull request closed without merging**